Week3/homework/step2-4.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 function Dog() {
-  // add your code here
+  this.name = 'Rupert';
+  this.color = 'brown';
+  this.numLegs = 4;
```

Comment:
You've hard-coded Rupert, brown, and 4, into the Dog constructor. That means any hound you create will always have these values.

The idea here is that Dog should take name, color, and numLegs as parameters - which you'd pass in when you create it. Like this;

const hound = new Dog('Rupert', 'brown', 4);